crates/next-custom-transforms/tests/fixture/server-actions/server-graph/33/output.js

```diff
@@ -2,7 +2,7 @@
 import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
 import { cache as $$cache__ } from "private-next-rsc-cache-wrapper";
 const v = 'world';
-export var $$RSC_SERVER_CACHE_0 = $$cache__("default", "803128060c414d59f8552e4788b846c0d2b7f74743", 0, async function fn() {
+export var $$RSC_SERVER_CACHE_0 = $$cache__("default", "803128060c414d59f8552e4788b846c0d2b7f74743", "fn (app/item.js:3:1)", 0, async function fn() {
```

Comment:
what would this look like if the cache function was in a node_modules installed by pnpm or yarn PnP? it might be good to transform those names into like a virtual filename like `(node_modules/specifier:line:col)`

Where specifier came from whatever was doing the importing when we discovered the "use cache". It's awkward b/c different specifiers could point to the same use cache function in a 3rd party dep but putting the million-character long folder name that sometimes shows up with package managers like pnpm might be bad too.

Even if we can't make it good in this PR (like if we want to land it first and solve this edge case later) we should at least make sure it doesn't blow up if you compile a "use cache" from a package manager